condominium/data/ir_ui_view.xml

```diff
@@ -0,0 +1,341 @@
+        <field name="model">res.partner</field>
+        <field name="priority">200</field>
+        <field name="type">kanban</field>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//kanban//templates//t/field[@name='country_id']" position="after">
+                    <field name="x_condominiums_ids" display="full"/>
+                </xpath>
+            </data>
+        </field>
+    </record>
+    <record id="res_partner_tree_custom" model="ir.ui.view">
+        <field name="name">res.partner.tree customization</field>
+        <field name="inherit_id" ref="base.view_partner_tree"/>
+        <field name="mode">extension</field>
+        <field name="model">res.partner</field>
+        <field name="priority">160</field>
+        <field name="type">tree</field>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//field[@name='user_id']" position="replace"/>
+                <xpath expr="//field[@name='company_id']" position="after">
+                    <field name="x_condominiums_ids" optional="show" widget="many2many_tags"/>
+                </xpath>
+            </data>
+        </field>
+    </record>
+    <record id="res_partner_select_custom" model="ir.ui.view">
+        <field name="name">res.partner.select customization</field>
+        <field name="inherit_id" ref="base.view_res_partner_filter"/>
+        <field name="mode">extension</field>
+        <field name="model">res.partner</field>
+        <field name="priority">160</field>
+        <field name="type">search</field>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//filter[@name='ubl_cii_format']" position="replace"/>
+                <xpath expr="//filter[@name='group_country']" position="replace"/>
+                <xpath expr="//filter[@name='salesperson']" position="replace"/>
+                <xpath expr="//field[@name='user_id']" position="replace"/>
+                <xpath expr="//field[@name='parent_id']" position="replace">
+                    <field name="x_condominiums_ids"/>
+                </xpath>
+            </data>
+        </field>
+    </record>
+    <record id="sale_order_form_custom" model="ir.ui.view">
+        <field name="name">sale.order.form customization</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
+        <field name="mode">extension</field>
+        <field name="priority">160</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//field[@name='has_recurring_line']" position="after">
+                    <button name="%(ir_act_server_split_per_property)d" string="Split per Property" type="action" invisible="not x_condominium_units_ids"/>
+                </xpath>
+                <xpath expr="//button[@name='action_sale_order_log']" position="after">
```

Comment:
For me it can be removed, as it can be in the header as well, no?
```suggestion
```

Reply:
If we remove it then it will show like this 
![image](https://github.com/odoo/industry/assets/120459793/ba983aa1-6c27-4837-b984-e75735e39296)



And we want to show it like this 
![image](https://github.com/odoo/industry/assets/120459793/3dd43d7f-e5f0-46fb-be60-7cc7fa30e461)